xml/admin_backup.xml

```diff
@@ -221,6 +222,12 @@ mysql --password=$(cat /var/lib/misc/infra-secrets/mariadb-root-password) velum_
      </para>
     <screen>&prompt.root;<command>reboot now</command></screen>
     </step>
+    <step>
+     <para>
+      Once the new &admin_node; has restarted, ssh back onto it and run the following command.
+<screen>&prompt.root;<command> docker exec -it $(docker ps -qf name="salt-master") salt-key -A</command></screen>
```

Comment:
you have to press 'y' to accept nodes, pls change to `docker exec -i $(docker ps -qf name="salt-master") salt-key -A -y` for answering 'Yes' from the command